### ISSUE TITLE
Add sample timeline to website

### ISSUE THREAD

**Issue by wslack** [usds/SME-QA-Website#147]
Content:

#### Example Timeline

- Week One: Job analysis workshop
- Week Two: Circulate workshop output, draft job announcement, begin recruitment
- Week Three: Post Job Announcement (until midnight after 100 applicants), SME resume review training
- Week Four: SMEs do resume review
- Week Five: HR reviews notes and sends tiebreakers to other SMEs
- Week Six: HR reviews notes, begins scheduling interviews, and trains SMEs doing Interview 1
- Weeks Seven and Eight: SMEs hold interview 1. HR reviews notes, schedules round 2 interviews, and trains SMEs doing Interview 2.
- Weeks Eight and Nine: SMEs hold interview 2; HR reviews notes
- Weeks Ten and Eleven: HR issues certs; hiring managers hold final placement interviews and tell HR which applicants they want
- Week Twelve: HR sends offers. Applicants confirm or decline within 48 hours. HR creates new certificates with next category for the HM to consider. Repeat process; once hiring manager is done, share certificate with other interested offices at the agency.

**Comment by wslack** [comment]
Closing in favor of #133
